readme/BREAKING_CHANGES_AMENDMENTS.markdown

```diff
@@ -362,4 +362,22 @@ We're trying to replace all ServiceProxyFactory usages in portal with class Snap
 ## Alternatives
 Use com.liferay.portal.kernel.module.service.Snapshot instead.
 ----
+```
+----
+
+# 169322529677c73dbd060ea11b64a9eee56415c9
+
+Missing breaking change
+
+Correct message should be:
+```
+LPD-16492 ObjectFields with Aggregation and Formula businessType should not be indexable
+    
+# breaking
+## What modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java
+Adding new validation for Aggregation and Formula ObjectField, preventing them to be indexed.
+## Why
+Since these types of ObjectField have its values got by runtime calculate, We have no proper support for them in the Elastic Search.
```

Comment:
Three things here: 
- I think it should be "runtime calculation" instead of "runtime calculate"
- ", we have no proper.." should start with lowercase
- Replace "Elastic Search" with "Elasticsearch"